tools/azure-sdk-tools/devtools_testutils/resource_testcase.py

```diff
@@ -67,12 +69,9 @@ def create_resource(self, name, **kwargs):
         if self.is_live and self._need_creation:
             self.client = self.create_mgmt_client(ResourceManagementClient)
             parameters = {"location": self.location}
-            if self.delete_after_tag_timedelta:
-                expiry = datetime.datetime.utcnow() + self.delete_after_tag_timedelta
-                parameters["tags"] = {"DeleteAfter": expiry.replace(microsecond=0).isoformat()}
+            expiry = datetime.datetime.utcnow() + self.delete_after_tag_timedelta
+            parameters["tags"] = {"DeleteAfter": expiry.replace(microsecond=0).isoformat()}
```

Comment:
Should we add these to the try block below with the other tags?

Reply:
This one doesn't need to be, the others do because when running locally those environment variables wont be set.

Reply:
Ok so the there is no try get env variable in python to allow for these to be empty if they aren't found?